multicluster/apis/multicluster/v1alpha1/multiclusterconfig_types.go

```diff
@@ -44,6 +44,9 @@ type MultiClusterConfig struct {
 	// The precedence about which IP address (internal or external IP) of Node is preferred to
 	// be used as the cross-cluster tunnel endpoint. if not specified, internal IP will be chosen.
 	GatewayIPPrecedence Precedence `json:"gatewayIPPrecedence,omitempty"`
+	// The type about which kind of IP (ClusterIP or PodIP) is preferred to be used as the
+	// multi-cluster Service's Endpoints. if not specified, ClusterIP will be chosen.
```

Comment:
What will happen if user changes the option and restarts mc-controller? If we do not support changing, add a comment for that.

Reply:
Add a new description `All member clusters should use the same type in a ClusterSet.`